common/config.js

```diff
@@ -75,6 +75,27 @@ function completeDirectorConfig(director) {
       cpi: info.cpi
     });
   }
+  let boshRateLimitEnabled = config.enable_bosh_rate_limit;
+  if (boshRateLimitEnabled) {
+    const maxWorkers = _.get(director, 'max_workers', 6);
+    const userCreateWorkers = _.get(director, 'policies.user.create');
+    const userUpdateWorkers = _.get(director, 'policies.user.update');
+    const userDeleteWorkers = _.get(director, 'policies.user.delete');
+    const autoWorkers = _.get(director, 'policies.scheduled.max_workers', (maxWorkers / 2));
+
+    if (userCreateWorkers === undefined || userUpdateWorkers === undefined || userDeleteWorkers === undefined) {
```

Comment:
can define defaults in bosh release so that this validation need not be done.

Reply:
Ok, I will make the change once the bosh release changes are done